File: server/groute/src/main/java/com/ssafy/groute/dto/Place.java
package com.ssafy.groute.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Place {
    private int id;
    private String name;
    private String type;
    private String lat;
    private String lng;
    private String zipCode;
    private String contact;
    private String address;
    private String description;
    private int themeId;
    private int areaId;
    private String img;
    private String userId;
    private double rate;
    private int heartCnt;

}

File: client/Groute/app/src/main/java/com/ssafy/groute/src/dto/Place.kt
package com.ssafy.groute.src.dto

data class Place(
    val address: String,
    val areaId: Int,
    val contact: String,
    val description: String,
    val heartCnt: Int,
    val id: Int,
    val img: String,
    val lat: String,
    val lng: String,
    val name: String,
    val rate: Float,
    val themeId: Int,
    val type: String,
    val userId: String,
    val zipCode: String
){
    constructor():this("",0,"","",0,0,"","","","",0F,0,"","","")
}